src/FNA3D_Driver_Vulkan.c

```diff
@@ -2269,7 +2292,7 @@ static uint8_t VULKAN_INTERNAL_IsDeviceSuitable(
 	uint8_t *deviceRank
 ) {
 	uint32_t queueFamilyCount, queueFamilyRank, queueFamilyBest;
-	SwapChainSupportDetails swapChainSupportDetails;
+	SwapChainSupportDetails swapchainSupportDetails;
```

Comment:
Nit: Totally agree with the new capitalization but we should probably do that separately to make this diff smaller.

Reply:
Won't this make the spelling inconsistent in the patch though?